docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs

```diff
@@ -88,5 +92,92 @@ public static void IidChangePointDetectorTransform()
             // 7       0       7.00    0.50    0.00
             // 7       0       7.00    0.50    0.00
         }
+
+        // This example creates a time series (list of Data with the i-th element corresponding to the i-th time slot). 
+        // IidChangePointDetector is applied then to identify points where data distribution changed using time series 
+        // prediction engine. The engine is checkpointed and then loaded back from disk into memory and used for prediction.
+        public static void IidChangePointDetectorPrediction()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Generate sample series data with a change
+            const int size = 16;
+            var data = new List<IidChangePointData>(size);
+            for (int i = 0; i < size / 2; i++)
+                data.Add(new IidChangePointData(5));
+            // This is a change point
+            for (int i = 0; i < size / 2; i++)
+                data.Add(new IidChangePointData(7));
+
+            // Convert data to IDataView.
+            var dataView = ml.CreateStreamingDataView(data);
+
+            // Setup IidSpikeDetector arguments
+            string outputColumnName = "Prediction";
+            string inputColumnName = "Value";
+            var args = new IidChangePointDetector.Arguments()
+            {
+                Source = inputColumnName,
+                Name = outputColumnName,
+                Confidence = 95,                // The confidence for spike detection in the range [0, 100]
+                ChangeHistoryLength = size / 4, // The length of the sliding window on p-values for computing the martingale score. 
+            };
+
+            // Time Series model.
+            ITransformer model = new IidChangePointEstimator(ml, args).Fit(dataView);
+
+            // Create a time series prediction engine from the model.
+            var engine = model.CreateTimeSeriesPredictionFunction<IidChangePointData, ChangePointPrediction>(ml);
+            for(int index = 0; index < 8; index++)
+            {
+                // Anomaly change point detection.
+                var prediction = engine.Predict(new IidChangePointData(5));
```

Comment:
I think this is not exactly right for demonstrating ChangePoint since it is supposed to identify when there has been a change in trend, as opposed to SpikeDetection.  By definition, we would want to run a prediction on a series of new data points and see if at any point in the series it detects something that is not just a spike, a change in the trend.